configuration/cardano/shelley-dummy-genesis.json

```diff
@@ -0,0 +1,41 @@
+{
+    "activeSlotsCoeff": 0.99,
+    "protocolMagicId": 838299499,
+    "systemStart": "2020-01-01T00:20:40Z",
+    "genDelegs": {},
+    "updateQuorum": 12,
+    "maxMajorPV": 25446,
+    "maxLovelaceSupply": 100000000,
+    "initialFunds": {},
+    "networkMagic": 403,
+    "networkId": "Testnet",
+    "epochLength": 21600,
+    "securityParam": 2160,
+    "slotLength": 20,
+    "slotsPerKESPeriod": 216000,
+    "maxKESEvolutions": 1080000,
```

Comment:
How were these chosen?

Reply:
They were take from a Shelley genesis file that was known to parse correctly.